src/styles/card.scss

```diff
@@ -0,0 +1,131 @@
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  justify-content: center;
+  justify-items: center;
+  gap: 46px 48px;
+
+  padding: 50px 40px;
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+@media (max-width: 1023px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (max-width: 768px) {
+  .card__container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 488px) {
+  .card__container {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  transition: all 0.3s ease;
```

Comment:
It is a good practice to specify the exact properties instead of `all`
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x